tests/benchmarks/test_snowflake.py

```diff
@@ -59,7 +59,8 @@ def test_read_from_snowflake(perma_table, connection_kwargs, small_client):
 
 
 @pytest.mark.skipif(
-    dask.__version__ < Version("2022.10.0"), reason="median not available in dask"
+    Version(dask.__version__) < Version("2022.10.0"),
+    reason="median not available in dask",
```

Comment:
```suggestion
    Version(dask.__version__) < Version("2022.10.0"),
    reason="No support for median in dask < 2022.10.0",
```

Nit pick: Can we switch the message to be the suggestion to be consistent with https://github.com/coiled/coiled-runtime/blob/32b6c48c75ce54bf915e1bbc353c08f98e031a6f/tests/benchmarks/h2o/test_h2o_benchmarks.py#L96-L99